packages/cactus-test-tooling/src/test/typescript/integration/quorum/quorum-test-ledger/constructor-validates-options.test.ts

```diff
@@ -1,4 +1,4 @@
-const tap = require("tap");
+import tap = require("tap");
```

Comment:
@yayoiukai2021 The import here also needs to be refactored similar to the other tests